changes/1746.housekeeping

```diff
@@ -0,0 +1,2 @@
+Replaced `OrderedDict` instance in `nautobot/models/racks.py#275` with a plain
+`dict` instance.
```

Comment:
Line breaks in change fragments translate to separate entries in the release-notes, so this needs to be on one line:

```suggestion
Replaced `OrderedDict` instance in `nautobot/models/racks.py#275` with a plain `dict` instance.
```

Reply:
Whoops, looks like this got reverted!

Reply:
I think I inadvertently pulled in changes from the wrong develop branch and it mucked things up a bit, apologies for that. Should be good to go now though.